cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -21,7 +21,14 @@
     {%- endfor %}
 
     {% for block in page.content -%}
-        {{ render_block.render(block, loop.index) }}
+        {% if block.block_type == 'conference_registration_form' %}
```

Comment:
Is every possible form type going to need their own hardcoded entry in page templates, or can this be made generic?